app/javascript/multi-stamp/components/PrimerPanelFilter.vue

```diff
@@ -20,9 +20,14 @@ export default {
     }
   },
   computed: {
-    primerPanels() { // Returns the mutual primer panels
+    // Returns the mutual primer panels. Aggregates primer panel names by plate
+    // using a Map of Arrays, then reduces each array (corresponding to
+    // a single plate primer panle subset) returning only the primer panel
+    // names found in every plate.
+    primerPanels() {
       const primerPanelsByPlate = new Map()
-      this.requestsWithPrimerPanel.forEach((requestWithPlate) => {
+      for (let i = 0; i < this.requestsWithPrimerPanel.length; i++) {
```

Comment:
for loops are much more faster than forEach and since we don't really need the extra computational complexity that forEach comes with, I think the trade off is in favour of for loops.